frontend/src/components/Terminal/Terminal.jsx

```diff
@@ -43,14 +69,13 @@ export default class Terminal extends React.Component {
     }
 
     rowRenderer = ({ index, key, style }) => {
+        const { searchText, currentSearchRow } = this.props;
+        const searchWords = searchText ? [searchText] : [];
         return (
             <pre key={ key } style={ style }>
                 <Highlighter
-                    highlightClassName="error"
-                    searchWords={ [
-                        "ERROR",
-                        "WARNING",
-                    ] }
+                    highlightClassName={index === currentSearchRow ? 'current-search-result' : ''}
```

Comment:
Color it orange if it is the current search result. If it is a search result on a different line, color it yellow.